# dev numbers. these are labeled on devices
dev_num:
  insert_la: 5
  extend_la_left: 0
  extend_la_right: 1
  dumper_spin_left: 4
  dumper_spin_right: 3

